.gitignore

```diff
@@ -66,3 +59,15 @@ notebooks/.ipynb_checkpoints
 cfme-qe-yamls
 rhci-qe-data
 
+# exclude conf
+conf/
```

Comment:
I would rather say `conf/*.yaml` because we want the templates to be there and our config files always have `.yaml` extension ...